patch/kconfig-exclusions

```diff
@@ -98,7 +98,6 @@ CONFIG_USB_EHSET_TEST_FIXTURE
 CONFIG_USB_ISIGHTFW
 CONFIG_USB_YUREX
 CONFIG_USB_ATM
-CONFIG_DMA_SHARED_BUFFER
```

Comment:
@olivier-singla , check out this build failure. https://sonic-jenkins.westus2.cloudapp.azure.com/job/common/job/linux-kernel-buster-build-pr/33/